lua/dbee/handler/conn.lua

```diff
@@ -188,8 +193,22 @@ function Conn:show_page(page)
     self.page_ammount = self.page_ammount - 1
   end
 
+  if self.remaining_time == nil then
```

Comment:
nit: this can be removed and refactor the code below by adding `or` operator:
```lua
vim.api.nvim_win_set_option(
    winid,
    "winbar",
    "%="
      .. "Finished in "
      .. string.format("%.3f", self.remaining_time or 0.0)
      .. "s | "
      .. tostring(page + 1)
      .. "/"
      .. tostring(self.page_ammount + 1)
  )
```

Reply:
resolved in [1c5f1d7](https://github.com/kndndrj/nvim-dbee/pull/36/commits/1c5f1d7a47aaa388a43839884498e6e30f48560e)